components/CircleButton.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
```

Comment:
You don't need to import 'React' if you're not using it anywhere, so just import `{ Component }`